docs/application/dotnet/guides/multimedia/media-controller.md

```diff
@@ -41,69 +50,96 @@ To enable your application to use the media controller functionality:
 
--   To update the metadata and playback information on the server side:
+-   To update the metadata and playback information on the server side, follow these steps:
     1.  Start the media control server using the `Start()` method of the [Tizen.Multimedia.Remoting.MediaControlServer](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.Remoting.MediaControlServer.html) class:
 
-        ```
+        ```csharp
         MediaControlServer.Start();
         ```
 
     2. Update the metadata or playback information using the `SetMetadata()` or `SetPlaybackState()` methods:
 
-        ```
+        ```csharp
         MediaControlServer.SetPlaybackState(MediaControlPlaybackState.Playing, currentPosition);
         ```
 
     3. When no longer needed, stop the media control server using the `Stop()` method:
 
-        ```
+        ```csharp
         MediaControlServer.Stop();
         ```
 
-- To retrieve the metadata and playback information on the client side:
+- To retrieve the metadata and playback information on the client side, follow these steps:
     1.  Create a new instance of the [Tizen.Multimedia.Remoting.MediaControllerManager](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.Remoting.MediaControllerManager.html) class:
 
-        ```
+        ```csharp
         var mediaControllerManager = new MediaControllerManager();
         ```
 
     2. Retrieve the currently-active controllers and select one:
 
-        ```
+        ```csharp
         var controller = mediaControllerManager.GetActiveControllers().First();
         ```
 
     3. Retrieve the metadata or playback information from the server using the `GetMetadata()`, `GetPlaybackState()`, or `GetPlaybackPosition()` methods.
 
         For example, to retrieve the playback state from the server:
 
-        ```
+        ```csharp
         Tizen.Log.info(LogTag, $"Playback state is {controller.GetPlaybackState()}");
         ```
 
-<a name="send_media"></a>
+## Updating and Retrieving Playlist
+
+To create a playlist from the server side, create a new instance of the [Tizen.Multimedia.Remoting.MediaControlPlaylist](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.Remoting.MediaControlPlaylist.html) class:
+
+    ```csharp
+    var playlist = new MediaControlPlaylist("MyFavorite");
+    ```
+
+You can also create playlist with Metadata using the following code:
+
+    ```csharp
+    var playlist = new MediaControlPlaylist("MyFavoriate",
+        new Dictionary<string, MediaControlMetadata>()
+        {
+            {"Song1", new MediaControlMetadata() {Author = "Someone"} },
+            {"Song2", new MediaControlMetadata()}
+        }
+    );
+    ```
+
+To retrieve the playlist and metadata information on the client side, use the following code:
+    ```csharp
+        controller.PlaylistUpdated += (s, e) =>
+        {
+            Tizen.Log.Info(LogTag, $"Updated mode : {e.Mode}, Name : {e.Name},
```

Comment:
Originally it was same line but it make horizontal scroll bar so I did split it.
Should I make it same line?

Reply:
No, I just mean to use "," and ";" without red rectangles.
In the image I attached, or if you see the diff view, you would see new "," and ";" are highlighted with red background unlike other  ","s and ";"s.

I'm not sure with the exact reason, but because of something, special characters are mixed with "," and ";" added later.
To remove the red rectangles, please copy and paste the "," and ";" from other somewhere in the code.

Reply:
I made it single line.